googletest/include/gtest/internal/gtest-param-util.h

```diff
@@ -564,7 +564,10 @@ class ParameterizedTestSuiteInfo : public ParameterizedTestSuiteInfoBase {
 
           test_param_names.insert(param_name);
 
-          test_name_stream << test_info->test_base_name << "/" << param_name;
+	  if (test_info->test_base_name.size() > 0)
```

Comment:
```suggestion
          if (!test_info->test_base_name.empty()) {
            test_name_stream << test_info->test_base_name << "/";
          }
          test_name_stream << param_name;
```

Reply:
Ups - forgot this - new version of the pr in progress..

Reply:
Note: This is incorporated in the last pull req.